content/acra/configuring-maintaining/general-configuration/acra-rollback.md

```diff
@@ -77,6 +77,217 @@ Zones are deprecated since 0.94.0, will be removed in 0.95.0.
+  * `false` — (default) validate leaf certificate and all intermediate certificates
+
+  This option may be enabled in cases when intermediate CAs are trusted and there is no need to verify them all the time.
+  Also, even if this flag is `false` but there is no CRL's URL configured and there is no CRL's URL in intermediate CA certificates,
+  these intermediate CAs won't be validated since we don't know which CRLs could be used for validation.
+
+* `--tls_ocsp_required=<policy>`
+
+  How to handle situation when OCSP server doesn't know about requested certificate and returns "Unknown".
+
+  * `denyUnknown` — (default) consider "Unknown" response an error, certificate will be rejected
+  * `allowUnknown` — reverse of `denyUnknown`, allow certificates unknown to OCSP server
+  * `requireGood` — require all known OCSP servers to respond "Good" in order to allow certificate and
+    continue TLS handshake, this includes all URLs validator can use, from certificate (if not ignored) and from configuration
+
+* `--tls_ocsp_url=<url>`
+
+  URL of OCSP service.
+  Empty by default.
+
+  Should point to HTTP server that accepts `application/ocsp-request` MIME type
+  and responds with `application/ocsp-response`.
+
+* `--tls_ocsp_from_cert=<policy>`
+
+  How to treat OCSP server URL described in a certificate itself.
+
+  * `use` — try URL(s) from certificate after the one from configuration (if set)
+  * `trust` — try URL(s) from certificate, if server returns "Valid", stop further checks
+  * `prefer` — (default) try URL(s) from certificate before the one from configuration (if set)
+  * `ignore` — completely ignore OCSP's URL(s) specified in certificate
+
+* `--tls_ocsp_check_only_leaf_certificate={true|false}`
+
+  This flag controls behavior of validator in cases when certificate chain contains at least one intermediate certificate.
+
+  * `true` — validate only leaf certificate
+  * `false` — (default) validate leaf certificate and all intermediate certificates
+
+  This option may be enabled in cases when intermediate CAs are trusted and there is no need to verify them all the time.
+  Also, even if this flag is `false` but there is no OCSP's URL configured and there is no OCSP's URL in intermediate CA certificates,
+  these intermediate CAs won't be validated since we don't know whom to ask about them.
+
+* `--tls_database_auth=<mode>`
+
+  Set authentication mode that will be used for TLS connection with a database.
+  Possible values are the same as for `--tls_auth`.
+  Overrides the `--tls_auth` setting.
+  Default is `-1` which means "take value of `--tls_auth`".
+
+* `--tls_database_key=<filename>`
+
+  Path to private key that will be used for TLS handshake with a database.
+  Should correspond to the certificate configured with `--tls_database_cert`.
+  Empty by default.
+
+* `--tls_database_cert=<filename>`
+
+  Path to client TLS certificate shown to database during TLS handshake (overrides `--tls_cert`).
```

Comment:
it wrong old description too, because it doesn't override `tls_cert` but use it if empty. If `tls_cert` will be defined and there is some other flag tls_*_cert, then `tls_database_cert` will not override `tls_cert` and other flag too.